src/stores/NotificationsController.ts

```diff
@@ -1,30 +1,39 @@
 import { logout, reloadActiveSubscription } from './AccountController';
 
 import useService from '#src/hooks/useService';
-import { addQueryParams } from '#src/utils/formatting';
+import { addQueryParams, removeQueryParam } from '#src/utils/formatting';
 import { simultaneousLoginWarningKey } from '#src/components/LoginForm/LoginForm';
+import { queryClient } from '#src/containers/QueryProvider/QueryProvider';
 
 export enum NotificationsTypes {
-  ACCESS_GRANTED = 'access.granted',
   ACCESS_REVOKED = 'access.revoked',
   CARD_REQUIRES_ACTION = 'payment.card.requires.action',
+  CARD_FAILED = 'payment.card.failed',
+  CARD_SUCCESS = 'payment.card.success',
   SUBSCRIBE_REQUIRES_ACTION = 'subscribe.requires.action',
   FAILED = '.failed',
   SUBSCRIBE_FAILED = 'subscribe.failed',
+  SUBSCRIBE_SUCCESS = 'subscribe.success',
   ACCOUNT_LOGOUT = 'account.logout',
 }
 
-export const subscribeToNotifications = async (uuid: string = '') => {
+export const subscribeToNotifications = async (uuid: string = '', navigateTo: ({ url }: { url: string }) => void) => {
```

Comment:
Can you just use the return type of useNavigate here?
```suggestion
export const subscribeToNotifications = async (uuid: string = '', navigateTo: NavigateFunction => void) => {
```

Reply:
that is a nice suggestion! thanks